Sources/ShopifyCheckout/CheckoutView.swift

```diff
@@ -68,6 +68,9 @@ class CheckoutView: WKWebView {
 		configuration.applicationNameForUserAgent = CheckoutBridge.applicationName
 
 		super.init(frame: frame, configuration: configuration)
+		if #available(iOS 16.4, *) {
+			self.isInspectable = true
+		}
```

Comment:
Not sure we want to ship this, though I do find myself adding it every time. Should we put it behind a `debug` flag in the configuration object?

Reply:
fair. removing